tests/test_ecs/test_ecs_boto3.py

```diff
@@ -1756,7 +1819,7 @@ def test_update_task_definition_family_through_cloudformation_should_trigger_a_r
     cfn_conn.update_stack(StackName="test_stack", TemplateBody=template2_json)
 
     ecs_conn = boto3.client("ecs", region_name="us-west-1")
-    resp = ecs_conn.list_task_definitions(familyPrefix="testTaskDefinition")
+    resp = ecs_conn.list_task_definitions(familyPrefix="testTaskDefinition2")
```

Comment:
Adding the filter broke this test due since it filters now. I needed to change the prefix value here since it does a full string match on the family